department-of-veterans-affairs/caseflow
# APPEALS-61414: Judge misspelled on Case Distribution Controls page

Resolves [APPEALS-61414](https://jira.devops.va.gov/browse/APPEALS-61414)

# Description
Updated seed and test data to fix a typo in a case distribution lever

## Acceptance Criteria

## Testing Plan
1. From a newly reseeded database or newly deployed demo environment:
  1. Login as a CDA control group admin
  2. Navigate to /case-distribution-controls
  3. Verify that the typo "jduge" does not exist in the AOJ AOD Affinity Days lever description
